### PR TITLE
add radio component and fix wallet screen

### PR DESCRIPTION
**Pull request by sanjanadhikari1**
- [ ] Develop the radio component according to design and added it in the storybook.
- [ ] Use new radio component in the updated Wallet Screen.
image

### CODE REVIEW

File: src/common/components/Radio/Radio.tsx [hunk 1]
```diff
@@ -1,20 +1,55 @@
...
 
-function Radio({ children, isChecked, clickHandler }: RadioProps): JSX.Element {
+const Radio = ({
+  children,
+  isChecked,
+  clickHandler,
+  className = 'w-5 h-5',
+  smallCircleClass = 'w-2 h-2',
+}: TRadioProps): JSX.Element => {
+  const [checked, setChecked] = useState<boolean>(isChecked)
```
Comment (CrazyOptimist) [inline src/common/components/Radio/Radio.tsx:19]: We do we need both isChecked from props and local checked?
I guess having only isChecking and clickHandler from props is enough

File: src/common/components/Radio/Radio.tsx [hunk 2]
```diff
@@ -1,20 +1,55 @@
...
+  className?: string
+  smallCircleClass?: string
   children: React.ReactNode
 }
 
-function Radio({ children, isChecked, clickHandler }: RadioProps): JSX.Element {
+const Radio = ({
+  children,
+  isChecked,
+  clickHandler,
```
Comment (CrazyOptimist) [inline src/common/components/Radio/Radio.tsx:15]: Better to have more conventional names, like in standard checkbox or radio inputs: `checked` and `onChange`.
Please update if that sounds good to you

File: src/common/components/Radio/Radio.tsx [hunk 3]
```diff
@@ -1,20 +1,55 @@
...
+    setChecked(isChecked)
+  }, [isChecked])
   return (
-    <Styles.Container className='radioContainer' onClick={clickHandler}>
-      {children}
-      <Styles.Input type='radio' checked={isChecked} onChange={() => null} />
-      <Styles.Span className='radioCheckmark' />
-    </Styles.Container>
+    <div className='flex items-center'>
+      <div
```
Comment (CrazyOptimist) [inline src/common/components/Radio/Radio.tsx:25]: radio input is better than a div: user can navigate to it using tab and toggle it using space

**Review by CrazyOptimist — CHANGES_REQUESTED** [review]

**Review by ngvtuan — APPROVED** [review]

File: src/common/components/Radio/Radio.tsx (hunk 1 above)
Comment (sanjanadhikari1) [reply src/common/components/Radio/Radio.tsx:19]: fixed

File: src/common/components/Radio/Radio.tsx (hunk 2 above)
Comment (sanjanadhikari1) [reply src/common/components/Radio/Radio.tsx:15]: fixed

File: src/common/components/Radio/Radio.tsx (hunk 3 above)
Comment (sanjanadhikari1) [reply src/common/components/Radio/Radio.tsx:25]: fixed

**Review by CrazyOptimist — APPROVED** [review]

**Pull request merged**